src/Nancy.ViewEngines.Razor/CSharp/CSharpRazorViewRenderer.cs

```diff
@@ -55,10 +58,18 @@ public CSharpRazorViewRenderer()
             this.ModelCodeGenerator = typeof(CSharpModelCodeGenerator);
 
             this.Provider = new CSharpCodeProvider();
+        }
+
+        /// <summary>
+        /// Get the host
+        /// </summary>
+        public NancyRazorEngineHost GetHost(string viewLocation)
+        {
+            var host = this.engineHostFactory.Create(new CSharpRazorCodeLanguage(), viewLocation);
 
-            this.Host = new NancyRazorEngineHost(new CSharpRazorCodeLanguage());
+            host.NamespaceImports.Add("Microsoft.CSharp.RuntimeBinder");

```

Comment:
Actually now see that the same namespace was imported before as well. I wonder what this affects on Mono :yum: